schema/test/unit/computed_columns/cif_user_full_name_test.sql

```diff
@@ -0,0 +1,38 @@
+begin;
+
+select plan(3);
+
+insert into cif.cif_user(id, uuid, given_name, family_name, email_address)
+  overriding system value
+  values
+    (1, '11111111-1111-1111-1111-111111111111', 'John', 'Doe', 'a1@abc.com'),
+    (2, '11111111-1111-1111-1111-111111111112', 'Nofamilyname', null, 'a2@abc.com'),
+    (3, '11111111-1111-1111-1111-111111111113', null, 'Nogivenname', 'a3@abc.com');
```

Comment:
I found a given name that refers to a family name (`Nofamilyname`0 a bit confusing on first read; maybe something like `Onlygivenname` instead?